lib/UserNotificationConfigHandler.php

```diff
@@ -108,7 +113,12 @@ public function markAboutToExpireNotificationSentFor(OldPassword $passInfo) {
 	 * to include the userid owning the password and an id for the password
 	 */
 	public function markExpiredNotificationSentFor(OldPassword $passInfo) {
-		$this->config->setUserValue($passInfo->getUid(), 'password_policy', 'expiredSent', $passInfo->getId());
+		$this->config->setUserValue(
```

Comment:
Please share the error which is fixed through this

Reply:
``$passInfo->getId()`` returns an ``int`` but ``setUserValue()`` expects a string.